onnx_tf/common/data_type.py

```diff
@@ -75,7 +75,8 @@ def any_dtype_to_onnx_dtype(np_dtype=None, tf_dtype=None, onnx_dtype=None):
 
 
 def tf_to_np_str(from_type):
-  return mapping.TENSOR_TYPE_TO_NP_TYPE[int(tf2onnx(from_type))].name
+  return mapping.TENSOR_TYPE_TO_NP_TYPE[int(
+      tf2onnx(from_type))].name if from_type != tf.bfloat16 else 'bfloat16'
```

Comment:
@chinhuang007 bfloat16 support is added in Opset 13 and tf-1.x branch only support up to Opset 12. bfloat16 should not be a type that user can use here.

Reply:
Removed bfloat16. Please review.